projects/plugins/jetpack/_inc/client/search/dashboard/module-control.jsx

```diff
@@ -166,8 +170,10 @@ function Search( props ) {
 								disabled={ isSavingEitherOption || ! props.hasActiveSearchPurchase }
 								onChange={ toggleInstantSearch }
 								toggling={ togglingInstantSearch }
+								className="is-search-admin"
 							>
-								{ __( 'Enable instant search experience (recommended)', 'jetpack' ) }
+								{ __( 'Enable instant search experience ', 'jetpack' ) }
+								<span>{ __( '(recommended)', 'jetpack' ) }</span>
```

Comment:
Why are we splitting the translation string into two here?

Reply:
Changed to use `createInterpolateElement` in 054dd30d16